src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConsistentStreamingSource.java

```diff
@@ -192,7 +192,7 @@ protected void getChanges2(ChangeEventSourceContext context, YBPartition ybParti
                         if (merger.isSlotEmpty(tabletId)) {
                             try {
                                 response = this.syncClient.getChangesCDCSDK(
-                                        table, streamId, tabletId, cp.getTerm(), cp.getIndex(), cp.getKey(),
+                                        table, streamId, tabletId, cp.getTerm(), cp.getIndex(), (cp.getKey() == null) ? "".getBytes() : cp.getKey(),
```

Comment:
will we be passing empty or null?

Reply:
Now we will be passing null, the handling will be done in the `GetCheckpointRequest.java` in the yb-client.